### ISSUE TITLE
max_tokens not defined as a valid parameter in CreateChatCompletionRequest

### ISSUE THREAD

**Issue by farouqaldori** [openai/openai-node#70]
### Describe the bug

It's perfectly fine to pass max_tokens to a chat completion request. 

It's documented, and I even tried it myself. 

Despite that, the types are not defined in the interface for CreateChatCompletionRequest

### To Reproduce

1. Perform a createChatCompletion request.
2. Pass the `max_tokens` parameter to your request.
3. You should see the warning:
`Object literal may only specify known properties, and 'max_tokens' does not exist in type 'CreateChatCompletionRequest'. `

### OS

macOS

### Node version

v16.17.0

### Library version

openai v3.2.0

**Comment by hbracy** [comment]
Same issue here!

**Comment by farouqaldori** [comment]
In the meantime, it's safe to use a TS-ignore.

E.g.

```
{
        temperature: 0.2,
        top_p: 1,
        frequency_penalty: 0,
        // @ts-ignore - max_tokens is a valid parameter, but the types are not up to date.
        max_tokens: 5,
        presence_penalty: 0,
        messages: []
}
```

**Comment by adamk22** [comment]
Created a PR that fixes this: https://github.com/openai/openai-node/pull/69

**Comment by schnerd** [comment]
Thank you! We'll publish a fix for this shortly.

**Comment by schnerd** [comment]
Published v3.2.1 – thanks for reporting!
